drizzlepac/haputils/wcs_graphics.py

```diff
@@ -216,91 +218,87 @@ def generate_graphic(wcs_dataDF, output_base_filename, log_level):
     num_of_datasets = len(wcs_dataDF.index)
     print('Number of datasets: {}'.format(num_of_datasets))
 
-    TOOLS = "box_zoom,wheel_zoom,box_select,lasso_select,reset,help"
-
-    # Define a figure object
-    # Aperture 1
-    p1 = figure(tools=TOOLS, toolbar_location="right")
-    p2 = figure(tools=TOOLS, toolbar_location="right")
-    p3 = figure(tools=TOOLS, toolbar_location="right")
-
-    # Get the existing/remaining columns of the WCS_COLUMNS to know which
-    # glyphs to generate
+    # TO DO: Alternate no longer works.  $name is not interpreted. See original code.
+    wcs_tips = [("Primary WCS", "@prim_wcsname"),
+                ("Alternate WCS", "@$name")]
+
+    # Instantiate the figure objects 
+    text = 'WCS Component Differences (Active - Alternate)'
+    figure1 = HAPFigure(title = text,
+                        x_label = 'Delta CRPIX1 (pixels)',
+                        y_label = 'Delta CRPIX2 (pixels)',
+                        grid_line_color = 'white',
+                        hover_tips = wcs_tips)
+    figure2 = HAPFigure(title = text,
+                        x_label = 'Delta CRVAL1 (pixels)',
+                        y_label = 'Delta CRVAL2 (pixels)',
+                        grid_line_color = 'white',
+                        hover_tips = wcs_tips)
+    figure3 = HAPFigure(title = text,
+                        x_label = 'Delta Scale (pixels/arcseconds)',
+                        y_label = 'Delta Orientation (degrees)',
+                        grid_line_color = 'white',
+                        hover_tips = wcs_tips)
 
     # Figure 1 delta_crpix1 vs delta_crpix2
     # Figure 2 delta_crval1 vs delta_crval2
-    wcs_components = ['del_def_wcsname', 'del_apriori_wcsname', 'del_apost_wcsname']
-    wcs_type_colors = ['blue', 'green', 'purple']
+    # Figure 3 delta_scale vs delta_orientation
     wcs_type_names = ['default', 'apriori', 'aposteriori']
```

Comment:
I have to wonder whether these should be module level variables (WCS_TYPE_NAMES?) instead of hidden, hard-coded options.